lib/notifier/queries.js

```diff
@@ -14,75 +14,82 @@ compliance - notifications related to warnings triggered by compliance settings
 error - notifications related to errors
 */
 
+function getMachineName (machineId) {
+  const sql = 'SELECT * FROM devices WHERE device_id=$1'
+  return db.oneOrNone(sql, [machineId])
+    .then(it => it.name)
+}
+
 const addNotification = (type, message, detail) => {
-    const sql = `INSERT INTO notifications (id, type, message, detail) values ($1, $2, $3, $4)`
-    return db.oneOrNone(sql, [uuidv4(), type, message, detail])
+  const sql = `INSERT INTO notifications (id, type, message, detail) values ($1, $2, $3, $4)`
```

Comment:
`values` should be capitalized, but it is not a problem
```suggestion
  const sql = `INSERT INTO notifications (id, type, message, detail) VALUES ($1, $2, $3, $4)`
```